src/views/SignupLedger.vue

```diff
@@ -104,23 +117,34 @@ export default class SignupLedger extends Vue {
         }
     }
 
+    private get statusScreenMessage() {
+        if (this.state !== SignupLedger.State.FETCHING_INCOMPLETE) return '';
+        else return 'We might have missed used addresses that have no balance.';
```

Comment:
I can't recall ever using "we" in user facing text. I suggest replacing it i.e.
```suggestion
        else return 'Used addresses without balance might have been missed.';
```